python/setup.py

```diff
@@ -19,10 +19,10 @@
         "pytest~=6.0",
         "pytest-xdist~=1.34",
         "scipy~=1.4",
-        "lib_logging",
+        #"lib_logging",
     ],
 
-    dependency_links=[
-        './../lib_logging#egg=lib_logging',
-    ],
+    #dependency_links=[
```

Comment:
Unless it causes something to break, I prefer that we leave an empty dependency_links list rather than removing it.